requirements-dev.txt

```diff
@@ -1,3 +1,3 @@
-coverage==7.2.7
+coverage==6.5.0
 mock==5.0.2
-responses==0.23.1
+responses==0.21.0
```

Comment:
Why these changes?

Reply:
Sorry, it was committed by mistake. Thank you for catching it. Reverted in 8acc40aef7dd326b0f5fd056aecf9babdfed36ca.